src/looper.js

```diff
@@ -387,12 +387,12 @@ class Score {
   constructor() {
     // for all of the arguments
     this.parts = [];
-    this.currentPart = new Array(arguments.length);
+    this.currentPart = 0;
 
     var thisScore = this;
     for (var i in arguments) {
       this.parts[i] = arguments[i];
-      this.parts[i].nextPart = this.parts[i + 1];
+      if (i > 0) this.parts[i - 1].nextPart = this.parts[i];
```

Comment:
```suggestion
      if (i > 0) {
        this.parts[i - 1].nextPart = this.parts[i];
      }
```

nice solution/bug fix here!

I prefer to use brackets but that's just my personal preference for if statements